src/src/facilitator/facilitator.service.ts

```diff
@@ -1738,18 +1741,14 @@ export class FacilitatorService {
 		  aadhar_token
 		  address
 		  aadhar_verified
-		  block_id
```

Comment:
are these fields removed on purpose from response?

Reply:
yes. they were unnecessary leading to slow speed of api